modules/ObjectControl/src/objectlistener.cpp

```diff
@@ -245,4 +256,21 @@ atos_interfaces::msg::Monitor createROSMessage(const MonitorMessage& monrMessage
 	msg.acceleration.accel.angular.y = 0;
 	msg.acceleration.accel.angular.z = 0;
 	return msg;
+}
+
+sensor_msgs::msg::NavSatFix createNavSatFixMessage(const struct timeval& tv, double origin[3], const atos_interfaces::msg::Monitor &monr) {
```

Comment:
Good to know that the compiler treats
```C++
void myFun(double var[123]) {
    ...
}
```
the same as
```C++
void myFun(double* var) {
    ...
}
```
which can produce annoying errors.

```suggestion
sensor_msgs::msg::NavSatFix createNavSatFixMessage(const struct timeval& tv, std::array<double,3> origin, const atos_interfaces::msg::Monitor &monr) {
```

Reply:
Missed this one, thanks :+1: